lineapy/graph_reader/session_artifacts.py

```diff
@@ -252,7 +252,7 @@ def _get_input_variable_sources(self, pred_nodes) -> Dict[str, Set[str]]:
 
     def _get_common_variables(
         self, curr_nc: NodeCollection, pred_nc: NodeCollection
-    ) -> Tuple[Set[str], Set[LineaID]]:
+    ) -> Tuple[List[str], Set[LineaID]]:
```

Comment:
is there a chance of duplication now that you are returning a list instead of set? do you want to return a list of unique inner vars or are repeats ok?

Reply:
In this function no, since we only convert the set to list and sort it at the return statement.